### ISSUE TITLE
How to insert image

### ISSUE THREAD

**Issue by AlirezaAhmadiRad** [onizet/html2openxml#18]
Hello

I have some tables in an HTML page and want some images "Anchored" to special cells, overlay on page (not inside of cell and not changing cell size). Image data is in format of base64 included in image tag.
 How can I develop HTML for that purpose.

Thanks

**Comment by onizet** [comment]
Hello,

image in base64 are supported in this library. Do you have a sample in HTML to share? I don't full understand what you mean by "anchored"

**Comment by AlirezaAhmadiRad** [comment]
Sorry for using unknown word "Anchor". This is the name used in Excel and open office for excel that refers to the way images are addressed. Location of images in excel page is dependant to cell at top left side of image.  something like: image starts at cell B10 + (5px horizontal, 2px vertical). This is the way we receive data from excel. 
Unfortunately I don't have any relevant html code because I use data from excel and calculate it in JS. You don't need to follow any special html from my side, please just tell me how is this possible to add an image on top of table which starts from a special cell of table.
Thnaks

**Comment by onizet** [comment]
I did a test but unfortunately, this is not possible in Word.
If you find a way, send me your document and I will take a closer look.

The way I work is simply using Word to generate the expected result. Then, using the [OpenXml SDK Tools](https://www.microsoft.com/en-us/download/details.aspx?id=30425), decompile the document and see the generated xml/code.

**Comment by AlirezaAhmadiRad** [comment]
Although I'm not familiar with openxml, I did some search and I found this [post](https://stackoverflow.com/questions/42898907/how-to-add-a-shape-in-word-document-using-openxml-c) containing key  examples. Here we have sample of using [Anchor object](https://msdn.microsoft.com/en-us/library/documentformat.openxml.drawing.wordprocessing.anchor%28v=office.14%29.aspx?f=255&MSPPError=-2147217396) in docx file. 
Inside Anchor, we have property of layoutInCell with the following explanation:

- When this attribute has a value of true, then the object shall be positioned within the existing table cell, causing the cell to be resized as needed. This means that all positioning shall be relative to the cell and not the line on which the table appears.

- When this attribute has a value of false, then the object shall be positioned as specified, but the table shall be resized and/or relocated within the document as needed to accommodate the object. This means that all positioning shall be relative to the line on which the table appears and not the cell in which the anchor is present.

Everything seems to be available.
I tried to feed a base64 image to docx file to try all these things but I was not successful. I'm trying to find a working example and get image shown in document, then I will think of anchoring.
